docs/components/drawer/index.html

```diff
@@ -124,4 +124,78 @@ <h3 v-text="title"></h3>
         </footer>
       </div>
     </section>
+
+    <section>
+        <header>
+          <h2 id="basic-drawer">Positionable Descendants</h2>
+        </header>
+  
+        <div class="example" id="vue-slottedDrawerDemo" >
+          <header>
+            <form class="beta-hxForm">
+              <hx-text-control>
+                <input
+                  id="txtSlottedDrawerTitle"
+                  type="text"
+                  v-model="drawerTitle"
+                />
+                <label for="txtSlottedDrawerTitle">
+                  Title
+                </label>
+              </hx-text-control>
+              <p><code>hx-drawer &gt; [slot="fixed"]</code></p>
+            </form>
+          </header>
+  
+          <div>
+              <hx-disclosure aria-controls="drawer-with-slotted-positionable-children" class="hxBtn">
+                  Drawer with Slotted Positionable Children
```

Comment:
Instead of `children`, please change to `element`.

Reply:
Changes have been done, please review.